frontend/templates/product-list/sections/FilterableProductsSection/Toolbar.tsx

```diff
@@ -70,7 +70,10 @@ export function Toolbar(props: ToolbarProps) {
                justify="flex-end"
             >
                <SearchInput
-                  placeholder={`Search ${getProductListTitle(productList)}`}
+                  placeholder={`Search ${getProductListTitle({
```

Comment:
Hmm...  how is this passing CI? This should fail a typescript check I think. The function takes normal arguments, but we're passing an object with named keys.

Reply:
Oh nevermind, I see the first arg is an object with keys. Carry on!